AR_avalanche_exploration.ipynb

```diff
@@ -139,9 +139,354 @@
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n",
+      "/tmp/ipykernel_15667/1451147953.py:22: FutureWarning: Series.bool is now deprecated and will be removed in future version of pandas\n",
+      "  if (avy_start_date <= ar_end_date).bool() & (ar_start_date <= avy_end_date).bool():\n"
+     ]
+    }
+   ],
    "source": [
     "#~2 min, brute force iteration\n",
     "#for each avalanche ID, create a date range of 1 week preceding the avalanche\n",
```

Comment:
Minor suggestion, not required. Just might be nice to ditch all of the FutureWarnings, e.g. with 

```
if (avy_start_date <= ar_end_date).all() & (ar_start_date <= avy_end_date).all():
```

or by ignoring with the `warnings` module.

Reply:
Thanks for the review, Kyle! I will supress those warnings in the notebook and then squash/merge 👍🏻